### PR TITLE
favicon add

### PR DESCRIPTION
**Pull request by EgleJuske**
favicon added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" type="image/png" href="./images/favicon/">
```
Comment (Jjurgita) [inline index.html]: Better add a favicon link after the title. I think all links should be in the same place

**Review by Jjurgita — CHANGES_REQUESTED** [review]
Comment was added in files changed section

File: index.html (hunk 1 above)
Comment (EgleJuske) [reply index.html]: good notice. Done 👍

**Review by Jjurgita — APPROVED** [review]
Great 👍

**Review by InetaVei — APPROVED** [review]

**Review by JurateJ — APPROVED** [review]

**Pull request merged**